demo/index.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { createStore } from 'redux';
+import { Provider, connect } from 'react-redux';
+import { render } from 'react-dom';
+import BrandwatchReactAuth from '../';
+
+const apiService = {
+  store: null,
+};
+
+const AUTH_RECEIVE_PROFILE = 'AUTH_RECEIVE_PROFILE';
+const authReceiveProfile = (payload) => ({ type: AUTH_RECEIVE_PROFILE, payload })
+const store = createStore((state, { type, payload }) => {
+  switch(type) {
+    case AUTH_RECEIVE_PROFILE:
+      return Object.assign({}, state, {
+        profile: payload.profile,
+      });
+    default:
+      return state;
+  }
+});
+
+class DemoContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      token: null,
+    };
+  }
+
+  componentWillMount() {
+    apiService.store.getToken({ aud: 'brandwatch.com' }).then((token) => {
+      this.setState({ token });
```

Comment:
Yup agreed... this is purely the demo application to show the authentication flow has worked.